code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -504,6 +506,11 @@ as performing this in action() will cause the upgrade to end up in the borg inst
 		R.module.basic_modules += SP
 		R.module.add_module(SP, FALSE, TRUE)
 
+		ADD_TRAIT(R, TRAIT_CHEMWHIZ, CYBORG_ITEM_TRAIT) // Added traits to medical proccessor so that Medical borgs can do medical stuff
+		ADD_TRAIT(R, TRAIT_RESEARCHER, CYBORG_ITEM_TRAIT)
+		ADD_TRAIT(R, TRAIT_MEDICALEXPERT, CYBORG_ITEM_TRAIT)
+		ADD_TRAIT(R, TRAIT_SURGERY_HIGH, CYBORG_ITEM_TRAIT)
+
 /obj/item/borg/upgrade/processor/deactivate(mob/living/silicon/robot/R, user = usr)
```

Comment:
You should probably make this method remove the trait as well.

Reply:
Done so, Tested, and works.